code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,238 @@
+		REMOVE_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+	for(var/mob/living/carbon/human/human_in_range in spiral_range(9,local_user))
+		if(IS_HERETIC(human_in_range) || IS_HERETIC_MONSTER(human_in_range))
+			continue
+
+		SEND_SIGNAL(human_in_range,COMSIG_VOID_MASK_ACT,rand(-2,-20)*delta_time)
+		if(DT_PROB(60,delta_time))
+			human_in_range.hallucination += 5
+
+		if(DT_PROB(40,delta_time))
+			human_in_range.Jitter(5)
+
+		if(DT_PROB(30,delta_time))
+			human_in_range.emote(pick("giggle","laugh"))
+			human_in_range.adjustStaminaLoss(10)
+
+		if(DT_PROB(25,delta_time))
+			human_in_range.Dizzy(5)
+
+/obj/item/melee/rune_knife
+	name = "Carving Knife"
+	desc = "Cold Steel, pure, perfect, this knife can carve the floor in many ways, but only few can evoke the dangers that lurk beneath reality."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "rune_carver"
+	flags_1 = CONDUCT_1
+	sharpness = SHARP_EDGED
+	w_class = WEIGHT_CLASS_SMALL
+	wound_bonus = 20
+	force = 10
+	throwforce = 5
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("attacks", "slashes", "stabs", "slices", "tears", "lacerates", "rips", "dices", "rends")
+	attack_verb_simple = list("attack", "slash", "stab", "slice", "tear", "lacerate", "rip", "dice", "rend")
+	///turfs that you cannot draw carvings on
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava))
+	///A check to see if you are in process of drawing a rune
+	var/drawing = FALSE
+	///A list of current runes
+	var/list/current_runes = list()
+	///Max amount of runes
+	var/max_rune_amt = 3
+	///Linked action
+	var/datum/action/innate/rune_shatter/linked_action
+
+/obj/item/melee/rune_knife/examine(mob/user)
+	. = ..()
+	. += "This item can carve 'Alert carving' - nearly invisible rune that when stepped on gives you a prompt about where someone stood on it and who it was, doesn't get destroyed by being stepped on."
+	. += "This item can carve 'Grasping carving' - when stepped on it causes heavy damage to the legs and stuns for 5 seconds."
+	. += "This item can carve 'Mad carving' - when stepped on it causes dizzyness, jiterryness, temporary blindness, confusion , stuttering and slurring."
+
+/obj/item/melee/rune_knife/Initialize()
+	. = ..()
+	linked_action = new(src)
+
+/obj/item/melee/rune_knife/pickup(mob/user)
+	. = ..()
+	linked_action.Grant(user, src)
+
+/obj/item/melee/rune_knife/dropped(mob/user, silent)
+	. = ..()
+	linked_action.Remove(user, src)
+
+/obj/item/melee/rune_knife/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!is_type_in_typecache(target,blacklisted_turfs) && !drawing && proximity_flag)
+		carve_rune(target,user,proximity_flag,click_parameters)
+
+/obj/item/melee/rune_knife/proc/carve_rune(atom/target, mob/user, proximity_flag, click_parameters)
+	var/obj/structure/trap/eldritch/elder = locate() in range(1,target)
```

Comment:
does this check for adjacent runes, or just runes on the same tile?

Reply:
adjacent